tests/test_track_check.py

```diff
@@ -266,10 +266,10 @@ def test_direction_continuity():
 @pytest.mark.parametrize(
     "vsi, vsi_previous, max_speed_change, expected",
     [
-        (12, 12, 12 / km_to_nm, 0),
-        (12, 12, (12 + 10.01) / km_to_nm, 10),
-        (12, 12, (12 + 9.99) / km_to_nm, 0),
-        (12, 12, None, 0),
+        (12 / km_to_nm, 12 / km_to_nm, 12 / km_to_nm, 0),
+        (12 / km_to_nm, 12 / km_to_nm, (12 + 10.01) / km_to_nm, 10),
+        (12 / km_to_nm, 12 / km_to_nm, (12 + 9.99) / km_to_nm, 0),
+        (12 / km_to_nm, 12 / km_to_nm, None, 0),
```

Comment:
@jjk-code-otter: Those test are failing. 

function `speed_continuity` has these inputs:
* vsi
* vsi_previous
* speeds
* max_speed_change

The testing function is called with speeds=`max_speed_change`  (3rd entry of the function call) and for `max_speed_change` in function `speed_continuty`  the default value of 10.0 is taken.

I think, a value for ``speeds`` is needed, isn't it?